libs/auth-api-lib/src/lib/delegations/delegations-incoming.service.ts

```diff
@@ -22,6 +26,10 @@ import { DelegationsIndexService } from './delegations-index.service'
 import { DelegationDTO } from './dto/delegation.dto'
 import { MergedDelegationDTO } from './dto/merged-delegation.dto'
 
+import type { Logger } from '@island.is/logging'
+
+const UNKNOWN_NAME = 'Óþekkt nafn'
```

Comment:
We have already this constant defined in few places in `delegations-incoming*` services. We should move it to some file in the constants folder and reuse it.

Reply:
Fixed.